packages/tools/devtools/devtools-browser-extension/src/messaging/Utilities.ts

```diff
@@ -13,31 +13,6 @@ function formatMessageForLogging(text: string, loggingOptions?: MessageLoggingOp
 	return `${loggingPreamble}${text}`;
 }
 
-/**
- * Relays the provided message to the window (globalThis).
- *
- * @remarks Thin wrapper to provide some message-wise type-safety, and to inject some automated logging.
- *
- * @internal
- */
-export function relayMessageToWindow<TMessage extends ISourcedDevtoolsMessage>(
```

Comment:
Inlined into `ContentScript.ts` since it was the only place this was used, and it made the stubbing easier.